mattermost/mattermost-server
# [MM-48727] Restore license checks for SSO in self-hosted deployments

#### Summary


Please see description of [JIRA ticket](https://mattermost.atlassian.net/browse/MM-48727).

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-48727
#### Release Note

```release-note
NONE
```

**QA Steps**

Please see QA steps in [JIRA ticket](https://mattermost.atlassian.net/browse/MM-48727).
